apps/studio/src/plugins/NotyPlugin.ts

```diff
@@ -21,10 +21,10 @@ const VueNoty = {
     return new Noty(params)
   },
 
-  show(text, type = 'alert', opts = {}) {
+  show(text, type = 'alert', opts: { escapeHtml?: boolean } = {}) {
```

Comment:
let's make this `allowRawHtml`, rathar than `escapeHtml`, then the check is `if (opts.allowRawHtml)`, which is better